src/AccessTokenHandler.ts

```diff
@@ -29,35 +28,21 @@ export class AccessToken implements LambdaInterface {
 		logger.setPersistentLogAttributes({});
 		logger.addContext(context);
 
-		switch (event.resource) {
-			case ResourcesEnum.TOKEN:
-				if (event.httpMethod === "POST") {
-					try {
-						logger.info("Received token request:", { requestId: event.requestContext.requestId });
-						return await AccessTokenRequestProcessor.getInstance(logger, metrics).processRequest(event);
-					} catch (error) {
-						logger.error({ message: "An error has occurred. ",
-							error,
-							messageCode: MessageCodes.SERVER_ERROR,
-						});
-						if (error instanceof AppError) {
-							return new Response(error.statusCode, error.message);
-						}
-						return new Response(HttpCodesEnum.SERVER_ERROR, "An error has occurred");
-					}
-				}
-				logger.warn("Method not implemented", { messageCode: MessageCodes.METHOD_NOT_IMPLEMENTED });
-				return new Response(HttpCodesEnum.NOT_FOUND, "");
-
-			default:
-				logger.error("Requested resource does not exist", {
-					resource: event.resource,
-					messageCode: MessageCodes.RESOURCE_NOT_FOUND,
-				});
-				return new Response(HttpCodesEnum.NOT_FOUND, "");
-
+		try {
+			logger.info("Received token request:", { requestId: event.requestContext.requestId });
```

Comment:
Know this wasn't you but since we're here shall we say something like `Received token request, starting AccessTokenRequestProcessor`? Like we have in the other handlers?

Reply:
Added new logs before triggering all our processors